lib/liquid/extensions.rb

```diff
@@ -25,6 +26,12 @@ def to_liquid
   end
 end
 
+class BigDecimal
+  def to_liquid
+    to_f
```

Comment:
what do you mean by "store"? The issue is that when converting strings to numbers (and they have a decimal point), we internally use BigDecimal. But if you chain filters together, this is kinda pointless... but I agree that this could be a separate PR. Would you prefer that I just do this convert in the `abs` filter? (like the other number filters do)